content/en/docs/setup/install/operator/index.md

```diff
@@ -93,6 +93,24 @@ EOF
 The controller will detect the `IstioOperator` resource and then install the Istio
 components corresponding to the specified (`demo`) configuration.
 
+If you have used `--watchedNamespaces` to watch custom namespace, make sure to specify `values.global.istioNamespace` as the following.
```

Comment:
@shamsher31 I would prefer to change it to this

If you have used `--watchedNamespaces` to initialize the istio operator, make sure the `IstioOperator` resource is located in the specified namespace list.

If you want to install istio control plane to in namespace other than `istio-system`, make sure to specify the namespace with `values.global.istioNamespace` as the following.

Reply:
Thanks, updated.